dotcom-rendering/src/components/YoutubeBlockComponent.importable.tsx

```diff
@@ -121,6 +122,13 @@ export const YoutubeBlockComponent = ({
 		abTestsApi?.isUserInVariant('IntegrateIma', 'variant') ?? false;
 	const abTestParticipations = abTests?.participations ?? {};
 
+	const [elementId, setElementId] = useState<number>();
```

Comment:
Is naming this elementId a little confusing as its a legacy concept that we are now removing. Isn't it the index?